bot/seasons/evergreen/snakes/utils.py

```diff
@@ -117,43 +112,54 @@
 
 
 def get_resource(file: str) -> List[dict]:
+    """Load Snake resources JSON."""
+
     with (SNAKE_RESOURCES / f"{file}.json").open(encoding="utf-8") as snakefile:
         return json.load(snakefile)
 
 
 def smoothstep(t):
-    """Smooth curve with a zero derivative at 0 and 1, making it useful for
-    interpolating.
-    """
+    """Smooth curve with a zero derivative at 0 and 1, making it useful for interpolating."""
+
     return t * t * (3. - 2. * t)
 
 
 def lerp(t, a, b):
     """Linear interpolation between a and b, given a fraction t."""
+
     return a + t * (b - a)
 
 
 class PerlinNoiseFactory(object):
-    """Callable that produces Perlin noise for an arbitrary point in an
-    arbitrary number of dimensions.  The underlying grid is aligned with the
-    integers.
-    There is no limit to the coordinates used; new gradients are generated on
-    the fly as necessary.
+    """
+    Callable that produces Perlin noise for an arbitrary point in an arbitrary number of dimensions.
+
+    The underlying grid is aligned with the integers.
+
+    There is no limit to the coordinates used; new gradients are generated on the fly as necessary.
+
+    Taken from: https://gist.github.com/eevee/26f547457522755cb1fb8739d0ea89a1
+    Licensed under ISC
     """
 
     def __init__(self, dimension, octaves=1, tile=(), unbias=False):
-        """Create a new Perlin noise factory in the given number of dimensions,
-        which should be an integer and at least 1.
-        More octaves create a foggier and more-detailed noise pattern.  More
-        than 4 octaves is rather excessive.
-        ``tile`` can be used to make a seamlessly tiling pattern.  For example:
+        """
+        Create a new Perlin noise factory in the given number of dimensions.
+
+        dimension should be an integer and at least 1.
+
+        More octaves create a foggier and more-detailed noise pattern.  More than 4 octaves is rather excessive.
+
+        ``tile`` can be used to make a seamlessly tiling pattern.
```

Comment:
```suggestion
        tile can be used to make a seamlessly tiling pattern.
```

Reply:
Same as before, prefer to keep wrapped in backticks.